locale/en.yml

```diff
@@ -5,7 +5,10 @@ flarum-nicknames:
     settings:
       max_label: Maximum nickname length
       min_label: Minimum nickname length
+      random_username_label: Randomize Usernames?
```

Comment:
Perhaps the labels shouldn't be questions, kinda odd as we don't do that elsewhere.